README.md

```diff
@@ -2,10 +2,43 @@
 
 The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
 
+The TSC is community and contributor driven and is a home for community managed 
+projects and assets. The mission of the Node.js Foundation is to enable and encourage 
+the adoption of Node.js and the TSC is the home of the community driven portion
+of executing that mission.
+
+Since the Foundation and the TSC are intended to support the community the size and scope
+of the work and projects in it will grow to accomodate the needs of the community.
+
```

Comment:
Ok, I see that you didn't intend to make the assumption though I interpreted it that way. I would suggest rewording "will grow to accommodate the needs of the community" to "may grow as needed to meet the needs of the community".

>What exactly are these resources? Other than mentorship, which is primarily on my shoulders early on, what exactly are we diluting? What resources are we allocating to these projects that we're in danger of running out of?

Even without us having collected a full list, I would assume that resources are not infinite :) I could guess: CI resources, marketing dollars, TSC staff time. I think we will learn this more clearly as we try to grow the scope.

> How do we establish the how of "how to support the ecosystem" other than to admit those that apply? 

How about inviting representative of potential member projects to the TSC meetings for some time, so we have a chance to learn from the perspectives of the community?